src/plugins/constdb2doc.c

```diff
@@ -157,17 +204,45 @@ void constdb2doc_itemdb(void)
 	fprintf(out_fp, "\n");
 }
 
+void merge_files()
```

Comment:
better read file line by line or whole file at once.

for read line by line, open file with mode "rt" and read by fgets, and write by fputs

for read whole file, use ftell to get size, allocated buffer with given size, then use fread for read whole file, and fwrite for write all at once.

Reply:
Thank you, for the solution. I updated.